src/WebJobs.Script.WebHost/WebHooks/DynamicWebHookReceiverConfig.cs

```diff
@@ -20,13 +22,8 @@ public Task<string> GetReceiverConfigAsync(string name, string id)
             // "id" will be the function name
             // we ignore the "name" parameter since we only allow a function
             // to be mapped to a single receiver
-            FunctionSecrets secrets = _secretManager.GetFunctionSecrets(id);
-            if (secrets != null)
-            {
-                return Task.FromResult(secrets.Key);
-            }
-
-            return null;
+            string functionSecret = _secretManager.GetFunctionSecrets(id).Values.FirstOrDefault();
```

Comment:
Why would a WebHook need to specify anything other than a key? When provisioning WebHooks with various keys, we can do the key ID lookup on our side. All we need to verify is that the specified code matches one of the configured keys, right?